docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -33,28 +48,30 @@ To verify ownership of your plugin, generate an Access Policy token that you'll
 
 1. [Create a Grafana Cloud account](https://grafana.com/signup).
 
-1. Login into your account and navigate to **My Account > Security > Access Policies**. Click **Create access policy**.
+1. Login into your account, and then go to **My Account > Security > Access Policies**. 
+
+1. Click **Create access policy**.
 
-   Realm: has to be your-org-name (all-stacks)
-   Scope: plugins:write
+   Realm: has to be **your-org-name** (all-stacks)
+   Scope: **plugins:write**
 
    ![Create access policy.](/img/create-access-policy-v2.png)
 
 1. Click **Create token** to create a new token.
 
-   The expiration date field is optional, though you should change tokens periodically for increased security.
+   **Expiration date** is optional, though you should change tokens periodically for increased security.
 
    ![Create access policy token.](/img/create-access-policy-token.png)
 
-1. Click **Create** and save a copy of the token somewhere secure for future reference.
+1. Click **Create**, and then save a copy of the token somewhere secure for future reference.
```

Comment:
We should maybe have a next steps here to present the choice of the subsequent sections (public/private)?

Reply:
agreed.